app/frontend/src/features/auth/email/ChangeEmail.tsx

```diff
@@ -58,6 +59,9 @@ export default function ChangeEmail(accountInfo: GetAccountInfoRes.AsObject) {
     <>
       <PageTitle>{CHANGE_EMAIL}</PageTitle>
       <>
+        <p>
+          {YOUR_EMAIL_IS} <pre>{accountInfo.email}</pre>
+        </p>
```

Comment:
```suggestion
        <Typography variant="body1">
          {YOUR_EMAIL_IS} <pre>{accountInfo.email}</pre>
        </Typography>
```
Use `Typography` from Material UI for consistency with the other stuff on the page